examples/pcf8574_i2c/main.cpp

```
@@ -0,0 +1,35 @@
+CharacterDisplayRenderer renderer(&lcdAdapter, 20, 4, 0xDF, 0xE1);
+LcdMenu menu(renderer);
+KeyboardAdapter keyboard(&menu, &Serial);
+
+void setup() {
+    Serial.begin(9600);
+    lcd.begin();
+    renderer.begin();
+    menu.setScreen(mainScreen);
+}
+
+void loop() {
```

cppcheck (2.10-2)

[style] The function 'loop' is never used.

(unusedFunction)